tests/test_evaluations/test_evaluation_functions.py

```diff
@@ -117,6 +117,31 @@ def test_evaluation_list_per_fold(self):
             self.assertIsNotNone(evaluations[run_id].value)
             self.assertIsNone(evaluations[run_id].values)
 
+    def test_evaluation_list_sort(self):
+        size = 10
+        task_id = 115
+        # Get all evaluations of the task
+        unsorted_eval = openml.evaluations.list_evaluations(
+            "predictive_accuracy", offset=0, task=[task_id])
+        # Get top 10 evaluations of the same task
+        sorted_eval = openml.evaluations.list_evaluations(
+            "predictive_accuracy", size=size, offset=0, task=[task_id], sort_order="desc")
+        self.assertEqual(len(sorted_eval), size)
+        self.assertGreater(len(unsorted_eval), 0)
+        sorted_output = []
+        unsorted_output = []
+        for eval in sorted_eval.values():
+            sorted_output.append(eval.value)
+        for eval in unsorted_eval.values():
+            unsorted_output.append(eval.value)
```

Comment:
Sorry for missing this the first time. Would you mind using a list comprehension instead here? i.e.:
```python
        sorted_output = [evaluation.value for evaluation in sorted_eval.values()]
        unsorted_output = [evaluation.value for evaluation in unsorted_eval.values()]
```
It's equivalent in function (but faster), and even being a bit more verbose in variable names (to avoid collision with the `eval` builtin), it fits perfectly fine in two lines :)
After that I think this looks good to go :+1: